build.fsx

```diff
@@ -618,7 +636,7 @@ Target "HelpMultiNodeTests" <| fun _ ->
 "CleanNuget" ==> "BuildRelease" ==> "Nuget"
 
 //docs dependencies
-"BuildRelease" ==> "Docs" ==> "AzureDocsDeploy" ==> "PublishDocs"
+"BuildRelease" ==> "Docs" ==> "AzureDocsDeploy" ==> "SourceBrowserDocs" ==> "PublishDocs"
```

Comment:
sourcebrowser should be independent of the code xml docs.

The dependency should be "BuildRelease" ==> "GenerateSourceBrowser" ==> "PublishSourceBrowser"